### PR TITLE
PULSSI-703 Root CA and client certificate creation instructions

### PR DESCRIPTION
**Pull request by sapajune**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,69 @@ local2.* /var/log/haproxy.log
 and restart rsyslog:
 
 `systemctl restart rsyslog.service`
+
+## Creating Private Root CA and CA Signed Client Certificate
+
+1. To create root CA, execute `bin/createRootCA.sh` and fill in data. Example:
+```
+[graylog-proxy]$./createRootCA.sh
+Root CA file [rootCA]:
```
Comment (kompa3) [inline README.md]: Is the generated rootCA.pem exactly the same file than what is needed later for docker container, ie. ca.pem?
* If yes, the names could be similar in this example

How do you handle the server dns certificate? Should you append it to ca.pem or what?

**Comment by sapajune** [comment]
rootCA.pem is needed only on server side, rootCA.key is not needed other than for signing - should I make this somehow clearer? Missing from documentation is where to get ca.pem for container. Not sure how should I document that.

**Comment by kompa3** [comment]
Ok it actually stands there: "Configure generated CA certificate as accepted CA on server side". Anyway, the documentation is clearer now except for one thing - where do I get ca.pem from?

**Comment by kompa3** [comment]
Please document the creation of server certificate chain pem file into https://wiki.digia.com/display/PULSSI/Creating+LB+cert+for+the+instance.

Also, server_ca.pem (or something) would be more descriptive name than ca.pem since there are now multiple CAs referred to.

**Pull request merged**